src/backend/distributed/citus--7.0-3--7.0-4.sql

```diff
@@ -0,0 +1,82 @@
+/* citus--7.0-3--7.0-4.sql */
+
+SET search_path = 'pg_catalog';
+
+CREATE TYPE noderole AS ENUM (
```

Comment:
Maybe add a comment that we need to replace the type if we want to add new roles.